docs/admin/kubeadm-join.md

```diff
@@ -0,0 +1,222 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Kubeadm join
+notitle: true
+---
+{% capture overview %}
+## Kubeadm join {#cmd-join}
+{% endcapture %}
+
+{% capture body %}
+{% include_relative _kubeadm/kubeadm_join.md %}
+
+### Join workflow 
+`kubeadm init` bootstraps a Kubernetes master node by executing the
+following steps:
+
```

Comment:
Did you intend to have two statements leading into the following step?
kubeadm init and kubeadm join